doc/man/scc-hypervisor-collector.1.md

```diff
@@ -0,0 +1,203 @@
+---
+title: SCC-HYPERVISOR-COLLECTOR
+section: 1
+header: SCC Hypervisor Collector
+date: June 2022
+---
+# NAME
+**scc-hypervisor-collector** - Collect & Upload Hypervisor details
+to SUSE Customer Center
+
+# SYNOPSIS
+
+**scc-hypervisor-collector [<optional>...]**
+
+# DESCRIPTION
+
+The **scc-hypervisor-collector** collects details that are relevant
+to SUSE Customer Center (SCC) subscription compliance tracking, from
```

Comment:
no comma after "tracking"

Reply:
Oops I may have cut and paste a bad example command - the input file to `nroff` should be the .1 file not the .1.md file; pandoc renders the *.X.md files to *.X... So yeah it will look wrong if you try to nroff the .md files ;-)

Reply:
Fixed

Reply:
this one is minor, but isn't related to my md file mistake :)